molecule/templates/test.j2

```diff
@@ -0,0 +1,2 @@
+this is a {{ test }}
+
```

Comment:
So PackageLoader won't let us break out of the molecule package. There are other loaders, like FileSystemLoader that will take a full path to a template, but that doesn't really test how we're using things. I don't really see a good option to having the test template coexist.